### ISSUE TITLE
Typo in Building a simple component tutorial

### ISSUE THREAD

**Issue by dhiachou** [emberjs/guides#1541]
I am sure that the line 153 of the file `source/localizable/tutorial/simple-component.md` should become : 
```
Since we want the image to be smaller at first, we will set the property to start as `false`:
``` 
instead of 
```
Since we do not want the image to be smaller at first, we will set the property to start as `false`:
````

**Comment by yawboakye** [comment]
Fixed in https://github.com/emberjs/guides/pull/1543. Issue can be closed.

**Comment by toddjordan** [comment]
Thanks @yawboakye and @dhiachou for catching/fixing!  Closing as fixed by #1543.
